### DISCUSSION TITLE
How to turn extensions back on with rigtools

### DISCUSSION THREAD

**Discussion by Go0gl** [3kh0/ext-remover#1197] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
after you turn something off like dyknow and you want to turn it back on (I have no idea why you'd want to) you can evaluate a random other extension to bring you to the page where you can turn stuff off and **on**

**Comment by Mds1gnal2** [comment]
Go to chrome://extensions, find the extension and flip the "allow file:/// handlers"
